collect_app/src/androidTest/java/org/odk/collect/android/formentry/FormFinalizingTest.java

```diff
@@ -0,0 +1,69 @@
+package org.odk.collect.android.formentry;
+
+import android.Manifest;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.activities.MainMenuActivity;
+import org.odk.collect.android.support.CopyFormRule;
+import org.odk.collect.android.support.ResetStateRule;
+import org.odk.collect.android.support.pages.MainMenuPage;
+import org.odk.collect.android.support.pages.SaveOrIgnoreDialog;
+
+@RunWith(AndroidJUnit4.class)
+public class FormFinalizingTest {
```

Comment:
@opendatakit/testers as you consider more regression checks to add to your list and/or the automated suite, please make sure you check these out. 😊